setup.py

```diff
@@ -33,8 +33,6 @@
 from distutils.command.sdist import sdist
 from distutils.errors import CompileError, LinkError
 
-from numpy.distutils.command.build_ext import build_ext
```

Comment:
Why not have `try: .... Except ImportError: raise RuntimeError('numpy needs to be installed to install scikit-image')`